### PR TITLE
[Languages] How to build a Chrome Extension that displays motivational quotes using Javascript.

### LINKED ISSUE #4129: [Languages] How to build a Chrome Extension that displays motivational quotes using Javascript.
### How to build a Chrome Extension that displays motivational quotes using Javascript. 

[Javascript] Developing a Chrome extension using Javascript.

Every Chrome Extension that exists on the chrome web store performs a  specific task. Over the years I have had to install a couple of extensions that either helped me accomplish tasks at work, or enabled me to maximize productivity. Having used the Pomodoro timer extension in the last few months, I noticed a steep increase in productivity so I thought to myself “I am going to build something similar” - A Chrome Extension that will display motivational quotes at scheduled intervals (cheesy right!). I had zero knowledge about how to build one until this point. Luckily, the official Chrome documentation helped me understand the concepts I needed to just get started with building what I wanted.

Chrome Extensions are relatively easy to develop, and yes you can create them locally on your machine and have them interact with any page on the web. Think of them as small applications residing on the browser and with their help we can access information about a website, and build awesome things with them.

In this tutorial, you the reader, and I are going to build a Chrome Extension using Javascript. its major functionality is to display random motivational quotes from an API. You would have learned about the core concepts used in this work, and this will just be sufficient to enable you to build something similar or even more complex. 

At the end of the article the reader should be able to :
- Build a basic Chrome extension
- Fetch data from an API at scheduled intervals  using a special-purpose API called 
  chrome.alarms
- Launch their extension on the chrome web store.
- Learn how to install locally on their machine or share it with their friends without necessarily publishing it in the Chrome Web Store.

#### Article quality
In comparison to other articles, I came across while trying to build this project I could not get enough support on how to make scheduled API calls (plain javascript doesn't do it). No article applied this. I am happy that people who want to schedule API calls in their extensions will find this article invaluable.
Also, most of the articles I read preferred to use a popup at the top right of the page. They also needed to click the extension icon for the popup to display. I opted for notifications that display at the bottom right corner instead(so it doesn't cause a distraction), the same place where Windows notifications are received on your desktop. You do not need to click an icon to get a notification. I automated it.

#### References
- https://developer.chrome.com/docs/extensions/mv3/getstarted/
- https://www.youtube.com/watch?v=7Tu2j2pc87I&t=418s
- https://www.youtube.com/watch?v=ew9ut7ixIlI&t=1s

#### Conclusion
The reader will learn how to build a Chrome Extension and make scheduled API calls using chrome.alarms

### PR DESCRIPTION
**Pull request by deverten**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [ ] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #4129 

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by geoffrey45** [comment]
Hello @deverten 👋
Welcome to the EngEd program. Thank you for submitting your first article. We will review your article for technical correctness, among other things, before it's published.

Please remove this file: `Muhammed-Umar/section-workspace.code-workspace`

**Comment by geoffrey45** [comment]
Hello @deverten 👋
Welcome to the EngEd program. Thank you for submitting your first article. We will review your article for technical correctness, among other things, before it's published.

Please remove this file: `Muhammed-Umar/section-workspace.code-workspace`

**Review by geoffrey45 — CHANGES_REQUESTED** [review]
Hello @deverten 

Welcome to the EngEd program. We will review your article for technical correctness, etc, before it gets published.

Please add code to your article as code blocks instead of images. Then remove the code images.

**Comment by Kayere** [comment]
Plagiarism check done ✔️

File: content/articles/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/index.md [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+### Prerequisites
+To better understand and build along with this tutorial you should have:
+
+- The Chrome browser installed.
+- An offline or online code editor (VS Code in my case).
+- Some knowledge of Javascript and its syntax.
+- Your desktops notification turned on - this is where our motivational quotes will be displayed.
+    
+### Table of Contents
+  - [Getting started](#Getting-started)
```
Comment (geoffrey45) [inline content/articles/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/index.md:25]: Please fix the table of contents. Links are not working

File: content/articles/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/index.md [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+- The `manifest_version` field tells us the current version we are building with. Ensure to use 3.0 as the previous version (2.0) is deprecated
+- Also, it is nice to start with a `version` that is small, you may want to update your extension as time passes. I go with 1.0 for now.
+
+Now that we have our manifest file setup, let us add the directory as an extension in developer mode in our Chrome browser.
+
+Navigate to the Extensions management page by clicking on the extensions menu button at the top right of the browser, and selecting manage extensions at the bottom of the menu.
+
+You should see a page like this:
+![developer](/engineering-education/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/images/developer-mode.PNG)
+
```
Comment (geoffrey45) [inline content/articles/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/index.md:73]: Uppercase letters are not supported in the article URL. ie. `How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript`. Please replace them with lowercase letters.

Fix all your images.

File: content/articles/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/index.md [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+Hooray🎉🎉 you have learned how to build a simple Chrome browser extension, register components like background script in the manifest file, fetch data from APIs, and learned to use Chrome extension APIs like chrome.alarms and chrome.notifications.
+
+You can now share with friends with or without hosting in the web-store.
+
+I'm excited to see the amazing things you'll build 🚀🚀
+
+### References
+- https://developer.chrome.com/docs/extensions/mv3/getstarted/
+- https://www.youtube.com/watch?v=7Tu2j2pc87I&t=418s
+- https://www.youtube.com/watch?v=ew9ut7ixIlI&t=1s
```
Comment (geoffrey45) [inline content/articles/How-to-build-a-Chrome-Extension-that-displays-motivational-quotes-using-Javascript/index.md:339]: Please name your references. for example, `Getting started - Chrome Developers`, etc

**Review by geoffrey45 — COMMENTED** [review]
@deverten Please check the comments below.

**Comment by geoffrey45** [comment]
@deverten Your pull request looks messy (195 files changed). Please close this pull request and open a new one with only your file changes. 

Mention me when you open the new pull request so that I can finish the review.

**Pull request closed without merging**